_posts/2024-05-23-get-involved-with-pyos.md

```diff
@@ -0,0 +1,90 @@
+
+## <i class="fa-solid fa-bullhorn"></i> Promote pyOpenSci online and in-person
+pyOpenSci is active on both [LinkedIn](https://www.linkedin.com/company/pyopensci) and [Fosstodon](https://fosstodon.org/@pyOpenSci), and you’re welcome to tag us on social media, as well as to [reach out with questions](mailto:media@pyopensci.org)! We also have [a series of brand assets and guidelines](https://docs.google.com/presentation/d/1glwf3BEPxo5H6pOHp6d55r2sbw8x-8kQZ9GzoLXwEDc/edit?usp=drive_link) if you need a copy of our logo, information on our fonts, and/or access to various illustrations we use.
+
+Dependent on timing, cost, and location, we’d also be more than happy to send you a set of pyOpenSci stickers to share! Please reach out to [media@pyopensci.org](mailto:media@pyopensci.org) to inquire about your event.
+
+Please feel free to use any information from [our website](https://www.pyopensci.org/) and our [pyOpenSci lightning talk template [currently WIP]](https://docs.google.com/presentation/d/1205l8y75SI9spdklKnxDZzwwjArHvpeIpwFGZOvVLjc/edit?usp=sharing), to share about the work pyOpenSci is doing. And don’t forget to share your personal experiences with pyOpenSci as well!
+
+## <i class="fa-solid fa-handshake-angle"></i> Volunteer with pyOpenSci
+If you're looking to be more hands-on with pyOpenSci, consider joining our Review Team or Editorial Board, digging into some code and working from our Help Wanted board, or writing a guest blog post!
+
+### pyOpenSci's Review Team
+pyOpenSci welcomes reviewers from a diversity of background and with varying levels of Python technical expertise. Some of the basic things that we look for in a reviewer include:
+
+* Familiarity with using the Python programming language.
+* Ability to evaluate a Python package for usability and documentation quality.
+* Ability to provide a technical review of Python package structure and code quality/approach to solving the problems that the package seeks to address.
+
+We like to have a mix of technical and usability focus in our reviews so it’s ok if you don’t have all of the above skills! And if you are interested in peer review but have never reviewed before, we offer a mentorship program where we will pair you up with someone who has more experience reviewing code. From this experience you can learn more and empower yourself with code review skills. Software review skills are generally useful in data science, so they are skills worth investing in!
+
+To join our review team, [fill out our reviewer form](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform). We will contact you if we have a package that we need reviewers for. It’s OK if you’ve never reviewed a package before! We’ll walk you through it.
+
+### pyOpenSci's Editorial Team
+The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
```

Comment:
maybe link tot he editor guide here instead? i just think a link (that rhymed 😆 ) is important to our guidebooks here!

you can also link to the editorial board listing on the website if people want to see who's already on the board.